lib/rules/class-order.js

```diff
@@ -0,0 +1,50 @@
+/**
+* @fileoverview Alphabetizes classnames.
+* @author Maciej Chmurski
+*/
+'use strict'
+
+// ------------------------------------------------------------------------------
+// Requirements
+// ------------------------------------------------------------------------------
+
+const utils = require('../utils')
+
+// ------------------------------------------------------------------------------
+// Helpers
+// ------------------------------------------------------------------------------
+
+// ------------------------------------------------------------------------------
+// Rule Definition
+// ------------------------------------------------------------------------------
+module.exports = {
+  meta: {
+    type: 'suggestion',
+    docs: {
+      url: 'https://eslint.vuejs.org/rules/class-order.html',
+      description: 'enforce classnames order',
+      category: undefined
+    },
+    fixable: 'code',
+    schema: []
+  },
+  create: context => {
+    return utils.defineTemplateBodyVisitor(context, {
+      "VAttribute[directive=false][key.name='class']" (node) {
+        const classList = node.value.value
+        const classListSorted = classList.split(' ').sort().join(' ')
```

Comment:
ASCII whitespace can be used to split classNames, Could you change it to support it?

https://infra.spec.whatwg.org/#ascii-whitespace

Reply:
I've used regex with `\s` this time